### ISSUE TITLE
Apps not working and no root. How do switch from lineage os to something else?

### ISSUE THREAD

**Issue by bitcoinmeetups** [waydroid/waydroid#1650]
Apps not working and no root. How do switch from Lineage OS to something else?

I need root and I need the apps to work. On Lineage OS the browser is questionable and there is no root so I can't use all functions in Termux.

I hope I can use some other distro instead of Lineage OS that seems to come preinstalled with waydroid.

**Comment by mrvictory1** [comment]
https://github.com/casualsnek/waydroid_script
This script can install Magisk (w/o zygisk) for root and ARM translation layers to make ARM based apps work. Which apps do not work? What are you trying to achieve?

**Comment by bitcoinmeetups** [comment]
Zypto doesn't work. Google Wallet doesn't work. How about making rooted Android fork available by default or by using a flag at installation? Or something like that.

**Comment by bitcoinmeetups** [comment]
I mean should I have to rely on a third party script made by someone called "casualsnek"? casual snake?

Ideally this should be an option baked into the main product instead. Oh by the way the browser in lineage also seemed to have some issues with the address bar. And the lineage main developer got into trouble pretty recently for making a spooky easter egg kind of recklessly. So Lineage is not ideal in itself plus it would be good if rooting was decided at waydroid init.

**Comment by bitcoinmeetups** [comment]
And opengapps.

**Comment by mrvictory1** [comment]
Creating images with specific features would increase permutations of images quite rapidly. There is no known way of bypassing SafetyNet (since 2024) on x86_64 Android to my knowledge, at very least you need a release-keys image which Waydroid doesn't provide. So don't expect Google Wallet to work. No idea about the other app.
The casualsnek script can install Magisk. It also claims to install zygisk but in my experience zygisk doesn't work. For zygisk I used (another fork)[https://github.com/mistrmochov/magiskWaydroid]. Termux w/ root works either way.
LineageOS is the gold standard custom ROM with a well established base and support for many devices, I don't see anything wrong with its inclusion.

**Comment by bitcoinmeetups** [comment]
No, no. I mean Lineage is better than nothing. Waydroid is an accomplishment. Maybe there's not much else available at the moment, I mean with regard to Lineage. Well let's see if something shows up in the future. 

Ok, so Google Wallet won't work. Hmm... Even with opengapps and certificate it won't work. Ok.

Well about the permutations, just one extra option - prerooted would be good, useful and kind of expected. 

That's it for today, I'm going to sleep.

Funny question:

Do clowns play chess?
